src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -761,7 +758,8 @@ static void create_rr_locked(grpc_exec_ctx *exec_ctx, glb_lb_policy *glb_policy,
     pp->wrapped_on_complete_arg.client_stats =
         grpc_grpclb_client_stats_ref(glb_policy->client_stats);
     if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-      gpr_log(GPR_INFO, "Pending pick about to PICK from 0x%" PRIxPTR "",
+      gpr_log(GPR_INFO,
+              "Pending pick about to (async) PICK from 0x%" PRIxPTR "",
```

Comment:
Agreed.  I vote for `%p` being the way to do it, since it's shorter and more readable.